maven-core/pom.xml

```diff
@@ -164,31 +164,6 @@ under the License.
             </excludes>
           </configuration>
         </plugin>
-        <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
```

Comment:
Is this change related to the issue somehow?

Reply:
It's obsolete as we no longer use create-timestamp goal.

Reply:
@slachiewicz Correct.